sw/airborne/modules/sensors/airspeed_ms45xx_i2c.c

```diff
@@ -221,13 +223,22 @@ void ms45xx_i2c_init(void)
 
 void ms45xx_i2c_periodic(void)
 {
+#ifdef USE_AIRSPEED_LOWPASS_FILTER
+  // Update the filter if needed
+  if(ms45xx.lowpass_tau != ms45xx.lowpass_tau_set) {
+    ms45xx.lowpass_tau_set = ms45xx.lowpass_tau;
+    init_butterworth_2_low_pass(&ms45xx_filter, ms45xx.lowpass_tau,
+                              MS45XX_I2C_PERIODIC_PERIOD, get_butterworth_2_low_pass(&ms45xx_filter));
+  }
+#endif
+
```

Comment:
maybe using a setting handler is a better way to update the time constant and call the init with the newly received value ?

Reply:
Do we have the guarantee that the telemetry callback and i2c_periodic are not executed in parallel in chibios?

Reply:
They are both part of the mainloop, only the low level transactions are using dedicated threads (accessed through protected buffers and transaction queues)

Reply:
Updated to use a handler